blackjack.rb

```diff
@@ -93,6 +105,13 @@ def player_hit_or_stay(current_hand)
     end
   end
 
+#cpu doesn't handle aces (unless I missed it).
+# I did something like this, but it assumes ace will auto change so you don't bust
+# if hand.any?{ |c| c.face == 'A' }
+#    hand.inject(:+) < 18
```

Comment:
sorry, you're right. This is not a very helpful comment. 
my code is similar to yours, except it draws cards using 'while' instead of 'until'.
so this is a method that is called in determining if cpu will draw a card. 
It basically says draw a card until your total is 16 or above, unless there are any aces, then draw until your total is 18 or above.
so for yours it would be something like:
>`def cpu_choice(currenthand)`
>  ` if currenthand.any?{ |c| c.face == 'A' }`
>     `   currenthand.hand_value >= 18`
> `else`
>   `  currenthand.hand_value >=16`
> `end`

call cpu_choice(dealah.hands_array[0]) instead of dealah.hands_array[0].hand_value >= 16

is that more helpful? 
again, it doesn't help all that much if aces can't go to 1 when you would bust at 11.